### PR TITLE
Updated schema

### PR DESCRIPTION
**Pull request by asa1997**

### CODE REVIEW

File: .vscode/simple_json.code-snippets
Comment (harimohanr) [inline .vscode/simple_json.code-snippets:1]: @asa1997 is this file required?

Comment (asa1997) [reply .vscode/simple_json.code-snippets:1]: @harimohanr Not required. It helps you in creating code snippets. For lilke json snippet or html table.

Comment (harimohanr) [reply .vscode/simple_json.code-snippets:1]: @asa1997 Remove the file and the .vscode directory if its not required. Add an entry in the gitignore for .vscode

Comment (asa1997) [reply .vscode/simple_json.code-snippets:1]: @harimohanr done

**Review by harimohanr — APPROVED** [review]
@asa1997 Looks okay
